app/components/Admin/NaicsCodeList.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import React from 'react';
+import {ListGroup} from 'react-bootstrap';
+
+interface Props {
+  naicsCodes: {code: string; description: string; id: string}[];
+}
+
+export const NaicsCodeList: React.FunctionComponent<Props> = ({naicsCodes}) => {
+  const router = useRouter();
+
+  return (
+    <ListGroup variant="flush">
+      {naicsCodes.map(({code, description, id}) => (
+        <Link
+          key={id}
+          prefetch={false}
+          href={{pathname: router.pathname, query: {naicsCodeId: id}}}
+        >
+          <ListGroup.Item active={router.query?.naicsCodeId === id}>
+            <b>{code}</b>
+            <br />
+            <small>{description}</small>
```

Comment:
Per the [design](https://youtrack.button.is/_persistent/Screen%20Shot%202021-03-22%20at%208.11.46%20AM.png?file=7-625&c=false&updated=1616427224019) could we swap the code number and description so the text label is above and in bold instead of the number?
I'd also recommend rendering the code below as regular body font-sized text instead of <small> because, while the user is likely scanning visually for the text, the code number still has similar importance to the description:

```
          <ListGroup.Item ...>
            <b>{description}</b>
            <br/>
            {code}
          </ListGroup.Item>
```